src/sidebar/components/annotation-omega.js

```diff
@@ -20,16 +24,24 @@ function AnnotationOmega({
   showDocumentInfo,
 }) {
   const createDraft = useStore(store => store.createDraft);
+  const setDefault = useStore(store => store.setDefault);
 
   // An annotation will have a draft if it is being edited
   const draft = useStore(store => store.getDraft(annotation));
+  const group = useStore(store => store.getGroup(annotation.group));
+
+  const isPrivate = draft ? draft.isPrivate : !isShared(annotation.permissions);
```

Comment:
The `util/permissions` module exports both `isPrivate` and `isShared` functions. Perhaps we could simplify the API of that module by removing the `isPrivate` function and just requiring callers to use `isShared(...)` everywhere like you're doing here.